pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -7,10 +7,32 @@ import { Metric, Axis, AxisUnits } from "src/views/shared/components/metricQuery
 import { GraphDashboardProps, nodeDisplayName, storeIDsForNode } from "./dashboardUtils";
 
 export default function (props: GraphDashboardProps) {
-  const { nodeIDs, nodesSummary, storeSources } = props;
+  const { nodeIDs, nodesSummary, storeSources, tooltipSelection } = props;
 
   return [
-    <LineGraph title="Ranges" sources={storeSources}>
+    <LineGraph 
+      title="Ranges" 
+      sources={storeSources}
+      tooltip={(
+        <div>
+          {`Range information for ${tooltipSelection}:`}
```

Comment:
Suggested: "The count of ranges in each state..." or something.

Also, don't include "for", because `tooltipSelection` is either "on node 1" or "across all nodes".